meta-networking/recipes-connectivity/snort/snort3_git.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "snort3 - a free lightweight network intrusion detection system for UNIX and Windows."
+HOMEPAGE = "http://www.snort.org/"
+SECTION = "net"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://COPYING;md5=78fa8ef966b48fbf9095e13cc92377c5"
+
+DEPENDS = "luajit libdaq libdnet flex hwloc libpcap libpcre zlib libtirpc"
+RDEPENDS:${PN} += "bash"
+
+SRC_URI = "git://github.com/snort3/snort3.git;protocol=https;branch=master"
+SRCREV = "e1760a8dbb829bb3fcf1a340ab6cc4bb80a47ecd" 
+
+inherit cmake pkgconfig
+
+S = "${WORKDIR}/git"
+
+EXTRA_OECONF = "\
+    --enable-static-daq \
```

Comment:
does this mean we get both shared and static libs? is the library under GPL-2.0 as well? if so then we should try to make static library generation as non-default.

Reply:
Yes It is under GPL-2.0 [snort3-license](https://github.com/snort3/snort3/blob/master/LICENSE) similar to libdaq. I removed these configurations as it is not needed for my use case.